core/extension/gdextension.cpp

```diff
@@ -828,16 +665,16 @@ void GDExtension::close_library() {
 }
 
 bool GDExtension::is_library_open() const {
-	return library != nullptr;
+	return loader != nullptr && loader->is_library_open();
```

Comment:
Again, I would have done:

```suggestion
	return loader.is_valid() && loader->is_library_open();
```

I'm not sure if checking `!= nullptr` works with `Ref<>`?

Reply:
I think this should work because `Ref<T>` implements the `!=` operator:

https://github.com/godotengine/godot/blob/6118592c6d88350d01f74faff6fd49754f84a7d0/core/object/ref_counted.h#L83-L88

But I agree about using the `is_valid`/`is_null` methods with `Ref<T>`, not sure how consistent we are in the codebase.